automation/pages/basePage.py

```diff
@@ -0,0 +1,20 @@
+class Page(object):
+
+    BaseURL = "http://127.0.0.1:8080"
+
+    def __init__(self, driver):
+        self.BaseURL = "http://127.0.0.1:8080"
+        self.driver = driver
+
+    def open(self):
+        URL = self.BaseURL + self.URL
+        self.driver.get(self.BaseURL + self.URL)
```

Comment:
There is string interpolation in python - https://www.python.org/dev/peps/pep-0498/#specification